src/index.js

```diff
@@ -289,10 +289,10 @@ const initialize = async () => {
       params: [
         {
           chainId: '0x64',
-          rpcUrls: ['https://dai.poa.network'],
-          chainName: 'xDAI Chain',
+          rpcUrls: ['https://rpc.gnosischain.com'],
```

Comment:
Hey @FrederikBolding, I wondering if we should change it to a local Ganache server running on a different port. So we have more control over this in our tests